### PR TITLE
Add sachin82's file

### PR DESCRIPTION
**Pull request by sachin82**
Pull request to review changes to title and arrow direction

### CODE REVIEW

**Comment by syon21** [comment]
@sachin82 : I will review this now...

**Comment by sachin82** [comment]
Thanks @syon21

File: _posts/0000-01-02-sachin82.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (syon21) [inline _posts/0000-01-02-sachin82.md:5]: Something new to consider for Line# 5
change "your text" to "my text"

**Review by syon21 — CHANGES_REQUESTED** [review]
@sachin82 : please change "your text" to "my text"

File: _posts/0000-01-02-sachin82.md (hunk 1 above)
Comment (sachin82) [reply _posts/0000-01-02-sachin82.md:5]: Thanks @syon21 . Let me check...

**Review by syon21 — APPROVED** [review]

**Pull request merged**